src/Core/Utils/ArgParser.cpp

```diff
@@ -4,6 +4,72 @@
 
 namespace obe::utils::argparser
 {
+    /// @brief Test string to see if it is a bool value
```

Comment:
Please use this style for Doxygen comments :) https://github.com/ObEngine/ObEngine/blob/c3095b1c2cb3fdcc2a2df04aae3052fb3cc1a5dd/include/Core/Animation/Animation.hpp#L121

Reply:
(Also, while it's nice that you took the time to do it, no need to document private/internal functions, especially for functions with a name as clear as those)

Reply:
Just removing comments from internal functions for ArgParser.